### PR TITLE
Iw-dev: add page iw: information link {dev, phy, reg, help examples}

### PR DESCRIPTION
**Pull request by pete3n**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): 5.19 **

### CODE REVIEW

File: pages/linux/iw-dev.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# iw dev
+
+> Show and manipulate wireless devices.
+> Channel, freq, reg information: <https://wireless.docs.kernel.org/en/latest/en/developers/documentation/channellist.html>.
+> More information: <https://wireless.wiki.kernel.org/en/users/Documentation/iw>.
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages/linux/iw-dev.md:5]:
```suggestion
> More information: <https://wireless.docs.kernel.org/en/latest/en/users/documentation/iw.html>.
```

File: pages/linux/iw.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # iw
 
 > Show and manipulate wireless devices.
-> More information: <https://manned.org/iw>.
+> See also: `iw dev`.
+> More information: <https://wireless.wiki.kernel.org/en/users/Documentation/iw>.
```
[the remark this answers is not in the document]
Comment (spageektti) [reply pages/linux/iw.md:5]:
```suggestion
> More information: <https://wireless.docs.kernel.org/en/latest/en/users/documentation/iw.html>.
```

File: pages/linux/iw-dev.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (pete3n) [reply pages/linux/iw-dev.md:5]: Sorry, this seems to have migrated to: https://wireless.docs.kernel.org/en/latest/en/users/documentation/iw.html

File: pages/linux/iw-dev.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+- Set device WiFi frequency in Mhz (device must first be in monitor mode with the interface up):
+
+`sudo iw dev {{wlp}} set freq {{freq_in_mhz}}`
+
+- Show all known station info:
+
+`iw dev {{wlp}} station dump`
+
+- Create virtual interface in monitor mode with a specific MAC address:
```
Comment (spageektti) [inline pages/linux/iw-dev.md:27]:
```suggestion
- Create a virtual interface in monitor mode with a specific MAC address:
```

File: pages/linux/iw.md [hunk 4]
```diff
@@ -18,3 +19,19 @@
...
+
+- List all wireless capabilities for all physical hardware interfaces:
+
+`iw phy`
+
+- List the kernel's current wireless regulatory domain information:
+
+`iw reg get`
+
+- Show help for all commands:
```
Comment (spageektti) [inline pages/linux/iw.md:35]:
```suggestion
- Display help for all commands:
```
style guide

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
LGTM, welcome to tldr and thanks for your contribution.

**Comment by spageektti** [comment]
Please do not resolve suggestions without telling why or making suggested changes.

**Comment by pete3n** [comment]
> Please do not resolve suggestions without telling why or making suggested changes.

Sorry, committed changes.

**Review by spageektti — APPROVED** [review]

**Pull request merged**